content/news/buzz/june-2024-status-update/contents.lr

```diff
@@ -0,0 +1,63 @@
+
+* We modified Briefcase to `distribute the stub binaries as standalone artefact, rather than including them in the app template <https://github.com/beeware/briefcase/pull/1849>`__. We also `significantly optimised how Briefcase uses templates <https://github.com/beeware/briefcase/pull/1855>`__. Between these two changes, the download size required for a first project on macOS has been reduced from around 200MB, to less than 4MB.
+
+* We `isolated Briefcase's use of Cookiecutter <https://github.com/beeware/briefcase/pull/1864>`__ so that all of Briefcase's cached content is contained in Briefcase's own cache folder.
+
+* We `added support for distribution certificates on macOS <https://github.com/beeware/briefcase/pull/1856>`__.
+
+* We `modified Toga's APIs to use namedtuple types for Size and Position references <https://github.com/beeware/toga/pull/2585>`__.
+
+* We corrected a number of stability issues with Toga's testbed CI, and `Added CI testing for Wayland on Linux <https://github.com/beeware/toga/pull/2670>`__.
+
+* We completed a `major overhaul of typing in Toga's API surface <https://github.com/beeware/toga/pull/2252>`__.
+
+* We `added support for conditional coverage in Toga's testing <https://github.com/beeware/toga/pull/2642>`__. It is now possible to get a clean coverage report without running the test suite on every Python version and every platform.
+
+* We `added an official API for customizing the system-installed menu items <https://github.com/beeware/toga/pull/2636>`__.
+
+* We `added an API for detecting the screens available for an app to use, and for setting the position of windows relative to those screens <https://github.com/beeware/toga/pull/1930>`__.
+
+* We `added support for "simple" apps <https://github.com/beeware/toga/pull/2649>`__ - apps that don't have menu bars.
+
+* We `added support for apps whose lifecycle isn't tied to a single "Main" window <https://github.com/beeware/toga/pull/2651>`__, such as background apps (apps that don't have any windows) and document-based apps.
+
+* We `added support for app modal dialogs <https://github.com/beeware/toga/pull/2669>`__.
+
+What's next?
+--------------
+
+Work on Toga's App interface isn't quite finished - we're expecting to finalise the Document API in the near future. Once that work is complete, our focus will be CPython binary packaging ecosystem. The work we've done so far this year has led to CPython 3.13 formally supporting iOS and Android as Tier 3 supported platforms; the next step is ensuring that the rest of the community can build iOS and Android packages. This will involve improvements to tools like ``pip``, ``cibuildwheel``, ``meson-python`` and more. We expect that this work will take most of the rest of the quarter to release; but there should be some initial signs of progress by the end of the month.
```

Comment:
```suggestion
Work on Toga's App interface isn't quite finished - we're expecting to finalise the Document API in the near future. Once that work is complete, our focus will be the mobile Python binary packaging ecosystem. The work we've done so far this year has led to CPython 3.13 formally supporting iOS and Android as Tier 3 supported platforms; the next step is ensuring that the rest of the community can build iOS and Android packages. This will involve improvements to tools like ``pip``, ``cibuildwheel``, ``meson-python`` and more. We expect that this work will take most of the rest of the quarter to release; but there should be some initial signs of progress by the end of the month.
```